src/main/Welcome.java

```diff
@@ -0,0 +1,549 @@
+package main;
+
+// Comment for adding purposes
+import java.awt.EventQueue;
+
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import java.awt.Font;
+
+import javax.swing.JButton;
+import javax.swing.SwingConstants;
+import java.awt.Color;
+import java.awt.event.ActionListener;
+import java.awt.event.ActionEvent;
+import javax.swing.JPanel;
+import java.awt.CardLayout;
+import javax.swing.JPasswordField;
+import javax.swing.JTextField;
+import javax.swing.JSlider;
+import javax.swing.JComboBox;
+
+public class Welcome {
+
+	private JFrame GUI;
+	private JTextField usernameField;
+	private JPasswordField passwordField;
+	private JTextField course1Field;
+	private JTextField course2Field;
+	private JTextField course3Field;
+	private JTextField course4Field;
+	private JTextField course5Field;
+	private JTextField course6Field;
+	private JTextField grade1Field;
+	private JTextField grade2Field;
+	private JTextField grade3Field;
+	private JTextField grade4Field;
+	private JTextField grade5Field;
+	private JTextField grade6Field;
+
+	/**
+	 * Launch the application.
+	 */
+	public static void main(String[] args) {
```

Comment:
For testing purposes, having a main method is alright, just remember to delete it and replace it in the launcher file (or whatever we are going to call it) before we submit.